docs/renderer.md

```diff
@@ -0,0 +1,61 @@
+---
+name: Renderer
+route: '/renderer'
+---
+
+# Render Engine
+
+## Overview
+
+React requires a rendering engine, typically when creating an application people use `react-dom`.
+When running tests, React still requires an engine. We currently support two different engines –
+`react-test-renderer` & `react-dom`. If you have both installed in your project, by default we will
+use `react-test-renderer`, assuming you're using the default imports:
+
+```js
+import { renderHook } from '@testing-library/react-hooks'
+```
+
+It does this because the library runs a check for available renderers, in the order:
+
+- `react-test-renderer`
+- `react-dom`
+
+If neither are available you will see an error asking you to check that one of the above is
+installed. If only one is installed, then it will use that renderer.
```

Comment:
The part here about how we auto detect and errors feels a bit too specific to our implementation details.  It probable enough to say we will auto detect a default renderer if either of the dependencies is installed and will use, `react-test-renderer` if both are installed.

Then maybe a note about why that's the default, i.e. fewer dependencies on the test environment (i.e. no DOM requirement) and works for hooks intended for both `react` and `react-native`.